features/items.ts

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import { MessageEmbed } from 'discord.js';
+import { region, basePath } from '../modules/constants';
+import { capitalizeWordsInString } from '../modules/cleanup';
+import { constructErrorMessage } from '../modules/messages/errorMessageGeneration';
+import { normalizeItemName, normalizeStatName,  } from '../modules/cleanup';
+import { itemMap } from '../modules/init';
+
+//Gets item ID from name
+export function getItemID(itemName: string): any {
+    const normalizedItemName: string = normalizeItemName(itemName);
+    return itemMap.get(normalizedItemName);
+} 
+
+//Item API call
+export async function makeItemAPICall(itemName: string): Promise<any> {
+    const itemID = getItemID(itemName);
+    return await axios.get(`${basePath}/${region}/items/${itemID}.json`)
+      .then((resp) => {
+        return resp.data;
+      });
+  }
+
+//Gets item and sends message to Discord
+export async function getItem(itemName: string): Promise<any> {
```

Comment:
Can you make this function name more clear? It's very close to `getItemID`. Like maybe `getItemDataFromAPI`.

Reply:
Changed to `fetchItem`.